react/Dockerfile

```diff
@@ -1,30 +1,19 @@
-FROM node:lts AS development
+## build stage ##
+FROM node:lts-alpine3.21 AS build
 
 WORKDIR /app
 
-COPY package*.json ./
-RUN npm ci
-
 COPY . .
 
-ENV CI=true
-ENV PORT=3000
-
-CMD [ "npm", "start" ]
-
-FROM development AS builder
+RUN npm install
 
 RUN npm run build
 
+## run stage ##
 FROM nginx:alpine
 
-# Copy config nginx
-COPY --from=builder /app/.nginx/nginx.conf /etc/nginx/conf.d/default.conf
-
-WORKDIR /usr/share/nginx/html
+COPY --from=build /app/dist /usr/share/nginx/html
 
-# Remove default nginx static assets
-RUN rm -rf ./*
+EXPOSE 80
 
-# Copy static assets from builder stage
-COPY --from=builder /app/dist .
+CMD ["nginx", "-g", "daemon off;"]
```

Comment:
This `CMD` already exists in the base image: https://github.com/nginx/docker-nginx/blob/eaf8875a1967d24cea6ed8b37109075e39ed9e43/stable/alpine-slim/Dockerfile#L108